src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,181 @@ tags: runtime-traditionalweb;
 
-1. Create a Local Integer Variable, named WizardStepIndex, to hold the current Wizard step.
+1. To display the **Previous** button when applicable, select the **Previous** button, right-click, and select **Enclose in If**.
 
-1. Drag two Button widgets and set their names to Continue and Back.
+    ![](images/wizard-image-26.png) 
 
-1. Create a new Screen Action named WizardNavigation.
+1. On the **Properties** tab, in the **Condition** property, enter the following:
 
-    ![](images/wizard-image-4.png)
+   `CurrentStep > 1`
 
-1. Create a mandatory boolean Input Parameter on this action named IsNext.
+    ![](images/wizard-image-17.png)
 
-1. Drag an Assign and set WizardStepIndex, to  `If(IsNext, WizardStepIndex + 1, WizardStepIndex - 1)`.
+    You have now created a condition that ensures that the user can never go below the number of steps in the wizard.
+       
+1. Repeat steps 13 and 14 for the **Next** button. Enter `CurrentStep < 4` as the **Condition** property. This ensures the **Next** button is only displayed when applicable, and the user can never go above the number of steps in the wizard.
 
-    ![](images/wizard-image-5.png)
+**Add content to the wizard**
 
-1. Drag an AjaxRefresh to refresh your content container on the screen.
+1. From the Toolbox, drag the If widget onto the main content area of your application's screen, just below the Wizard Pattern.
 
-1. Set the OnClick of the Continue Button to the action WizardNavigation with Ajax Submit as the Method. Set the IsNext parameter to True.
+    ![](images/wizard-image-30.png)
+
+1. On the **Properties** tab, in the **Condition** property, enter the following:
+
+   `CurrentStep = 1`
+           
+    ![](images/wizard-image-19.png)
+
+    You have now created a condition that controls what is displayed on screen when the step is acive.
+
+1. Repeat steps 1 and 2 for all of the Wizard Items. Replace **Condition** property with 2, 3, and 4 respectively.
+
+1. In the top section (True) of If widgets, create the relevant content for each of the steps.
+
+     The following example shows a Shipping Details step that contains labels and input boxes.
+
+      ![](images/wizard-image-18.png)
+
+**Create the wizard navigation**
+
+To ensure that all of the information the user enters is passed from step to step, you must create a screen action.
+
+
+
+) for the **Previous** and **Next** navigation buttons.
+
+1. Select the **Next** button.
+
+1. On the **Properties** tab, from the **Destination** drop-down, select **New Screen Action**.
+
+    ![](images/wizard-image-22.png)
+
+1. From the Toolbox, drag the Assign block onto your screen and set the **Variable** property to ``CurrentStep`` and the Value property to ``CurrentStep + 1``. This ensures that user inputs get passed from step to step.
+
+    ![](images/wizard-image-21.png)
+
+1. Repeat steps 2 and 3 for the **Previous** button.
```

Comment:
This isn't fully accurate - as for the Previous button you need to set it to "Current Step - 1"